bee-api/bee-rest-api/src/endpoints/routes/api/v1/info.rs

```diff
@@ -73,23 +76,35 @@ pub(crate) async fn info<B: StorageBackend>(
     Ok(warp::reply::json(&SuccessBody::new(InfoResponse {
         name: node_info.name.clone(),
         version: node_info.version.clone(),
-        is_healthy: health::is_healthy(&tangle, &peer_manager).await,
-        network_id: network_id.0,
-        bech32_hrp,
-        min_pow_score: protocol_config.minimum_pow_score(),
-        messages_per_second: 0f64,            // TODO
-        referenced_messages_per_second: 0f64, // TODO
-        referenced_rate: 0f64,                // TODO
-        latest_milestone_timestamp,
-        latest_milestone_index: *latest_milestone_index,
-        confirmed_milestone_index: *tangle.get_confirmed_milestone_index(),
-        pruning_index: *tangle.get_pruning_index(),
+        status: StatusResponse {
+            is_healthy: health::is_healthy(&tangle, &peer_manager).await,
+            latest_milestone_timestamp,
+            latest_milestone_index: *latest_milestone_index,
+            confirmed_milestone_index: *tangle.get_confirmed_milestone_index(),
+            pruning_index: *tangle.get_pruning_index(),
+        },
+        protocol: ProtocolResponse {
+            network_name: network_id.0,
+            bech32_hrp,
+            min_pow_score: protocol_config.minimum_pow_score(),
+            rent_structure: RentStructureResponse {
+                vbyte_cost: 0,     // TODO
```

Comment:
```suggestion
                vbyte_cost: 500,     // TODO
```
default value here https://github.com/Wollac/protocol-rfcs/blob/protocol-parameters/tips/TIP-0022/tip-0022.md

Reply:
Oh makes sense, was using these here https://github.com/gohornet/hornet/blob/764e4ee25995b1557129277a97dea244b923513e/core/protocfg/params.go#L39-L41